packages/client/tiptap/extensions/database/TextCell.tsx

```diff
@@ -0,0 +1,25 @@
+import * as Y from 'yjs'
+import {Input} from '../../../ui/Input/Input'
+import {ColumnId, RowId} from './data'
+import {useCell} from './hooks'
+
+export const TextCell = ({
+  doc,
+  rowId,
+  columnId
+}: {
+  doc: Y.Doc
+  rowId: RowId
+  columnId: ColumnId
+}) => {
+  const [value, setValue] = useCell(doc, rowId, columnId)
```

Comment:
-1 I pasted the entire Great Gatsby into a cell, which is a couple megabytes. Should we put a max length on the Input? Applies to the other cell types, too.